### PR TITLE
ColorToStringConverters

### PR DESCRIPTION
**Pull request by bijington**
Documentation added for .NET MAUI Community Toolkit proposals:

- [Proposal] `ColorToStringConverters ` <https://github.com/CommunityToolkit/Maui/issues/54>

### CODE REVIEW

**Comment by PRMerger15** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger12** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger19** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/converters/color-to-cmyk-string-converter.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+				converter: new ColorToCmykStringConverter()));
+
+		Content = new VerticalStackLayout
+		{
+			Children =
+			{
+				new Label { Text = "My favourite Color is:" },
+				label
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-cmyk-string-converter.md:71]:
```suggestion
	label.SetBinding(
		Label.TextProperty,
		new Binding(
			nameof(ViewModel.MyFavoriteColor),
			converter: new ColorToCmykStringConverter()));

	Content = new VerticalStackLayout
	{
		Children =
		{
			new Label { Text = "My favourite Color is:" },
			label
		}
	};
```

File: docs/maui/converters/color-to-cmyk-string-converter.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+			{
+				new Label()
+					.Text("My favourite Color is:"),
+				new Label()
+					.Bind(
+						Label.TextProperty,
+						nameof(ViewModel.MyFavoriteColor),
+						converter: new ColorToCmykStringConverter())
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-cmyk-string-converter.md:99]:
```suggestion
	{
		Children =
		{
			new Label()
				.Text("My favourite Color is:"),
			new Label()
				.Bind(
					Label.TextProperty,
					nameof(ViewModel.MyFavoriteColor),
					converter: new ColorToCmykStringConverter())
		}
	};
```

File: docs/maui/converters/color-to-cmyka-string-converter.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+				converter: new ColorToCmykaStringConverter()));
+
+		Content = new VerticalStackLayout
+		{
+			Children =
+			{
+				new Label { Text = "My favourite Color is:" },
+				label
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-cmyka-string-converter.md:71]:
```suggestion
	label.SetBinding(
		Label.TextProperty,
		new Binding(
			nameof(ViewModel.MyFavoriteColor),
			converter: new ColorToCmykaStringConverter()));

	Content = new VerticalStackLayout
	{
		Children =
		{
			new Label { Text = "My favourite Color is:" },
			label
		}
	};
```

File: docs/maui/converters/color-to-cmyka-string-converter.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+			{
+				new Label()
+					.Text("My favourite Color is:"),
+				new Label()
+					.Bind(
+						Label.TextProperty,
+						nameof(ViewModel.MyFavoriteColor),
+						converter: new ColorToCmykaStringConverter())
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-cmyka-string-converter.md:99]:
```suggestion
	{
		Children =
		{
			new Label()
				.Text("My favourite Color is:"),
			new Label()
				.Bind(
					Label.TextProperty,
					nameof(ViewModel.MyFavoriteColor),
					converter: new ColorToCmykaStringConverter())
		}
	};
```

File: docs/maui/converters/color-to-hex-rgb-string-converter.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+				converter: new ColorToHexRgbStringConverter()));
+
+		Content = new VerticalStackLayout
+		{
+			Children =
+			{
+				new Label { Text = "My favourite Color is:" },
+				label
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-hex-rgb-string-converter.md:71]:
```suggestion
	label.SetBinding(
		Label.TextProperty,
		new Binding(
			nameof(ViewModel.MyFavoriteColor),
			converter: new ColorToHexRgbStringConverter()));

	Content = new VerticalStackLayout
	{
		Children =
		{
			new Label { Text = "My favourite Color is:" },
			label
		}
	};
```

File: docs/maui/converters/color-to-hex-rgb-string-converter.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+			{
+				new Label()
+					.Text("My favourite Color is:"),
+				new Label()
+					.Bind(
+						Label.TextProperty,
+						nameof(ViewModel.MyFavoriteColor),
+						converter: new ColorToHexRgbStringConverter())
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-hex-rgb-string-converter.md:99]:
```suggestion
	{
		Children =
		{
			new Label()
				.Text("My favourite Color is:"),
			new Label()
				.Bind(
					Label.TextProperty,
					nameof(ViewModel.MyFavoriteColor),
					converter: new ColorToHexRgbStringConverter())
		}
	};
```

File: docs/maui/converters/color-to-hex-rgba-string-converter.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+				converter: new ColorToHexRgbaStringConverter()));
+
+		Content = new VerticalStackLayout
+		{
+			Children =
+			{
+				new Label { Text = "My favourite Color is:" },
+				label
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-hex-rgba-string-converter.md:71]:
```suggestion
	label.SetBinding(
		Label.TextProperty,
		new Binding(
			nameof(ViewModel.MyFavoriteColor),
			converter: new ColorToHexRgbaStringConverter()));

	Content = new VerticalStackLayout
	{
		Children =
		{
			new Label { Text = "My favourite Color is:" },
			label
		}
	};
```

File: docs/maui/converters/color-to-hex-rgba-string-converter.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+			{
+				new Label()
+					.Text("My favourite Color is:"),
+				new Label()
+					.Bind(
+						Label.TextProperty,
+						nameof(ViewModel.MyFavoriteColor),
+						converter: new ColorToHexRgbaStringConverter())
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-hex-rgba-string-converter.md:99]:
```suggestion
	{
		Children =
		{
			new Label()
				.Text("My favourite Color is:"),
			new Label()
				.Bind(
					Label.TextProperty,
					nameof(ViewModel.MyFavoriteColor),
					converter: new ColorToHexRgbaStringConverter())
		}
	};
```

File: docs/maui/converters/color-to-hsl-string-converter.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+				converter: new ColorToHslStringConverter()));
+
+		Content = new VerticalStackLayout
+		{
+			Children =
+			{
+				new Label { Text = "My favourite Color is:" },
+				label
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-hsl-string-converter.md:71]:
```suggestion
	label.SetBinding(
		Label.TextProperty,
		new Binding(
			nameof(ViewModel.MyFavoriteColor),
			converter: new ColorToHslStringConverter()));

	Content = new VerticalStackLayout
	{
		Children =
		{
			new Label { Text = "My favourite Color is:" },
			label
		}
	};
```

File: docs/maui/converters/color-to-hsl-string-converter.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+			{
+				new Label()
+					.Text("My favourite Color is:"),
+				new Label()
+					.Bind(
+						Label.TextProperty,
+						nameof(ViewModel.MyFavoriteColor),
+						converter: new ColorToHslStringConverter())
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-hsl-string-converter.md:99]:
```suggestion
	{
		Children =
		{
			new Label()
				.Text("My favourite Color is:"),
			new Label()
				.Bind(
					Label.TextProperty,
					nameof(ViewModel.MyFavoriteColor),
					converter: new ColorToHslStringConverter())
		}
	};
```

File: docs/maui/converters/color-to-hsla-string-converter.md [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+				converter: new ColorToHslaStringConverter()));
+
+		Content = new VerticalStackLayout
+		{
+			Children =
+			{
+				new Label { Text = "My favourite Color is:" },
+				label
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-hsla-string-converter.md:71]:
```suggestion
	label.SetBinding(
		Label.TextProperty,
		new Binding(
			nameof(ViewModel.MyFavoriteColor),
			converter: new ColorToHslaStringConverter()));

	Content = new VerticalStackLayout
	{
		Children =
		{
			new Label { Text = "My favourite Color is:" },
			label
		}
	};
```

File: docs/maui/converters/color-to-hsla-string-converter.md [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+			{
+				new Label()
+					.Text("My favourite Color is:"),
+				new Label()
+					.Bind(
+						Label.TextProperty,
+						nameof(ViewModel.MyFavoriteColor),
+						converter: new ColorToHslaStringConverter())
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-hsla-string-converter.md:99]:
```suggestion
	{
		Children =
		{
			new Label()
				.Text("My favourite Color is:"),
			new Label()
				.Bind(
					Label.TextProperty,
					nameof(ViewModel.MyFavoriteColor),
					converter: new ColorToHslaStringConverter())
		}
	};
```

File: docs/maui/converters/color-to-rgb-string-converter.md [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+				converter: new ColorToRgbStringConverter()));
+
+		Content = new VerticalStackLayout
+		{
+			Children =
+			{
+				new Label { Text = "My favourite Color is:" },
+				label
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-rgb-string-converter.md:71]:
```suggestion
	label.SetBinding(
		Label.TextProperty,
		new Binding(
			nameof(ViewModel.MyFavoriteColor),
			converter: new ColorToRgbStringConverter()));

	Content = new VerticalStackLayout
	{
		Children =
		{
			new Label { Text = "My favourite Color is:" },
			label
		}
	};
```

File: docs/maui/converters/color-to-rgb-string-converter.md [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+			{
+				new Label()
+					.Text("My favourite Color is:"),
+				new Label()
+					.Bind(
+						Label.TextProperty,
+						nameof(ViewModel.MyFavoriteColor),
+						converter: new ColorToRgbStringConverter())
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-rgb-string-converter.md:99]:
```suggestion
	{
		Children =
		{
			new Label()
				.Text("My favourite Color is:"),
			new Label()
				.Bind(
					Label.TextProperty,
					nameof(ViewModel.MyFavoriteColor),
					converter: new ColorToRgbStringConverter())
		}
	};
```

File: docs/maui/converters/color-to-rgba-string-converter.md [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+				converter: new ColorToRgbaStringConverter()));
+
+		Content = new VerticalStackLayout
+		{
+			Children =
+			{
+				new Label { Text = "My favourite Color is:" },
+				label
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-rgba-string-converter.md:71]:
```suggestion
	label.SetBinding(
		Label.TextProperty,
		new Binding(
			nameof(ViewModel.MyFavoriteColor),
			converter: new ColorToRgbaStringConverter()));

	Content = new VerticalStackLayout
	{
		Children =
		{
			new Label { Text = "My favourite Color is:" },
			label
		}
	};
```

File: docs/maui/converters/color-to-rgba-string-converter.md [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+			{
+				new Label()
+					.Text("My favourite Color is:"),
+				new Label()
+					.Bind(
+						Label.TextProperty,
+						nameof(ViewModel.MyFavoriteColor),
+						converter: new ColorToRgbaStringConverter())
+			}
+		};
```
Comment (jfversluis) [inline docs/maui/converters/color-to-rgba-string-converter.md:99]:
```suggestion
	{
		Children =
		{
			new Label()
				.Text("My favourite Color is:"),
			new Label()
				.Bind(
					Label.TextProperty,
					nameof(ViewModel.MyFavoriteColor),
					converter: new ColorToRgbaStringConverter())
		}
	};
```

**Review by jfversluis — COMMENTED** [review]
Code formattingggg!

**Comment by PRMerger18** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Review by jfversluis — APPROVED** [review]

**Pull request merged**